.github/workflows/release.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
           token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
           ref: edge
           path: docs
-          persist-credentials: false
+          # persist-credentials: false
```

Comment:
@DariuszPorowski Is this a temporary workaround? Curous why we commented it out instead of removing

Reply:
Yes, its temp. Final goal is to have it but it requires to refactor script attached to this workflows.